src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const summing = (number1) => (number2) => {
+    counter++;
+
+    return counter > 3 && counter % 2 === 0
+      ? 'Bzzz... Error!' : number1 + number2;
```

Comment:
all ternary variables should start on new line
```suggestion
    return counter > 3 && counter % 2 === 0
      ? 'Bzzz... Error!' 
      : number1 + number2;
```